go.mod

```diff
@@ -1,9 +1,18 @@
 module code.cloudfoundry.org/cfhttp
 
-go 1.16
+go 1.17
 
 require (
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.12.0
+	github.com/onsi/ginkgo/v2 v2.1.3
+	github.com/onsi/gomega v1.18.1
+)
+
+require (
+	github.com/golang/protobuf v1.5.2 // indirect
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
```

Comment:
Do we really need this import? The version used seems to be not official. Most probably because the PR is from the beginning of March 2022 and there haven't been official release of the library.

Perhaps it is worth updating the PR (go modules) with latest dependencies used.

Reply:
I have added a bump to latest direct and indirect dependencies in #28